### PR TITLE
updated to corda 4, including the checked in quasar.jar

### PR DESCRIPTION
**Pull request by nimmaj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,21 @@ See https://docs.corda.net/getting-set-up.html.
...
+	
+We recommend editing your IntelliJ preferences so that you use the gradle runner - this means that the quasar utils 
+plugin will make sure that these flags are set for you.
+
+Currently in `Build, Execution, Deployment -> Build Tools -> Gradle -> Runner` (but just search for `runner`)
+
+ * set "Delegate IDE build/run actions to gradle" to true
+ * set "Run test using:" to "Gradle Test Runner"
+
+Alternatively, you can:
```
Comment (anthonykeenan) [inline README.md]: "Alternatively, to use the built in IntelliJ test runner, you can:"

Is that clearer?

File: workflows/build.gradle [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 apply plugin: 'net.corda.plugins.quasar-utils'
 
 cordapp {
-    targetPlatformVersion 4
-    minimumPlatformVersion 4
+    targetPlatformVersion corda_platform_version.toInteger()
+    minimumPlatformVersion corda_platform_version.toInteger()
     workflow {
-        name "Template Flows"
+        name "Tempmlate Flows"
```
Comment (joeldudley) [inline workflows/build.gradle]: Typo here

File: README.md [hunk 3]
```diff
@@ -16,6 +16,21 @@ See https://docs.corda.net/getting-set-up.html.
 
 # Usage
 
+## Running tests inside IntelliJ
+	
+We recommend editing your IntelliJ preferences so that you use the gradle runner - this means that the quasar utils 
+plugin will make sure that these flags are set for you.
+
+Currently in `Build, Execution, Deployment -> Build Tools -> Gradle -> Runner` (but just search for `runner`)
```
Comment (joeldudley) [inline README.md]: Maybe specify this is under "Preferences" (macOS) or "Settings" (Windows/Linux)?

File: README.md [hunk 4]
```diff
@@ -16,6 +16,21 @@ See https://docs.corda.net/getting-set-up.html.
 
 # Usage
 
+## Running tests inside IntelliJ
+	
+We recommend editing your IntelliJ preferences so that you use the gradle runner - this means that the quasar utils 
+plugin will make sure that these flags are set for you.
```
Comment (joeldudley) [inline README.md]: What are "these flags" you're referring to? Confused me a little.

File: README.md [hunk 5]
```diff
@@ -16,6 +16,21 @@ See https://docs.corda.net/getting-set-up.html.
...
+
+Currently in `Build, Execution, Deployment -> Build Tools -> Gradle -> Runner` (but just search for `runner`)
+
+ * set "Delegate IDE build/run actions to gradle" to true
+ * set "Run test using:" to "Gradle Test Runner"
+
+Alternatively, you can:
+
+ * copy your quasar.jar to the `<project root>/lib/` dir
```
Comment (joeldudley) [inline README.md]: Which quasar.jar is "your" quasar.jar? Do they have to download it online? How do they pick a version?

**Review by joeldudley — CHANGES_REQUESTED** [review]
Hey Ben, thanks - much needed! Just a few comments :)

File: workflows/build.gradle (hunk 2 above)
Comment (nimmaj) [reply workflows/build.gradle]: ah - sorry - sorted

File: README.md (hunk 3 above)
Comment (nimmaj) [reply README.md]: Awesome idea - done something similar to what you suggested.

File: README.md (hunk 4 above)
Comment (nimmaj) [reply README.md]: Updated - is the new version ok?

File: README.md (hunk 5 above)
Comment (nimmaj) [reply README.md]: Updated a bit - do you think it's enough?

File: README.md (hunk 1 above)
Comment (nimmaj) [reply README.md]: Yes, much clearer.  Changed.

**Review by joeldudley — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -16,6 +16,26 @@ See https://docs.corda.net/getting-set-up.html.
 
 # Usage
 
+## Running tests inside IntelliJ
+	
+We recommend editing your IntelliJ preferences so that you use the gradle runner - this means that the quasar utils 
+plugin will make sure that some flags (like -javaagent - see "Alternatively..." below) are set for you.
+
+To switch to using the gradle runner, Currently in  (but just search for `runner`)
```
Comment (joeldudley) [inline README.md]: This line doesn't make sense any more

**Review by joeldudley — CHANGES_REQUESTED** [review]

**Comment by joeldudley** [comment]
@nimmaj Thanks Ben. Just one final request.

File: README.md (hunk 6 above)
Comment (nimmaj) [reply README.md]: OMG thank god you lot exist - sorry about that!

**Comment by nimmaj** [comment]
blimey - sorry about that - hopefully this version ok...

**Review by joeldudley — APPROVED** [review]
LGTM!

**Review by anthonykeenan — APPROVED** [review]

**Pull request merged**